pkg/reconciler/labeler/labeler_test.go

```diff
@@ -190,18 +242,28 @@ func simpleConfig(namespace, name string) *v1alpha1.Configuration {
 	cfg.Status.InitializeConditions()
 	cfg.Status.SetLatestCreatedRevisionName(name + "-dbnfd")
 	cfg.Status.SetLatestReadyRevisionName(name + "-dbnfd")
+
+	for _, opt := range opts {
+		opt(cfg)
+	}
 	return cfg
 }
 
-func simpleRevision(namespace, name string) *v1alpha1.Revision {
+func simpleRevision(namespace, name string, opts ...RevisionOption) *v1alpha1.Revision {
```

Comment:
it's not *simple* anymore, probably just `revision`.

Reply:
🙄 done